### ISSUE TITLE
How to check mAP in training?

### ISSUE THREAD

**Issue by canyilmaz90** [jwchoi384/Gaussian_YOLOv3#54]
@jwchoi384  Hi,
It's stated that we should check mAP in the training in several issues (https://github.com/jwchoi384/Gaussian_YOLOv3/issues/19#issuecomment-553878638) which asks why average loss is not converging during training. But, I didn't understand how to do it. Is there a hyperparameter in the config file? If not, how does it work? Does not the network try to converge according to loss function?

**Comment by jwchoi384** [comment]
@canyilmaz90 
The average loss is converging during training. 
Printed loss value in our code is not actual loss value, because of the gradient of uncertainty.
I didn't modify the calculation of loss value.
If you want to print the actual loss value, I think you should modify it.
https://github.com/jwchoi384/Gaussian_YOLOv3/blob/02b91a555211008cd362bd57d4384e883d7bd65d/src/gaussian_yolo_layer.c#L279
  
I just calculated the mAP with trained weights after training.
There is no hyper parameter related to this in cfg.

**Comment by canyilmaz90** [comment]
@jwchoi384 thanks for the explanation.
